core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -561,7 +561,8 @@ class GroupCoordinator(val brokerId: Int,
 
             // if this is the leader, then we can attempt to persist state and transition to stable
             if (group.isLeader(memberId)) {
-              info(s"Assignment received from leader for group ${group.groupId} for generation ${group.generationId}")
+              info(s"Assignment received from leader for group ${group.groupId} for generation ${group.generationId}. " +
+                s"The group has ${group.allMembers.size} members, ${group.allStaticMembers.size} of which are static.")
```

Comment:
Could you replace `${group.allMembers.size}` by `${group.size}`?

Reply:
Done.